operator/controllers/syncer/csi_syncer.go

```diff
@@ -559,7 +559,7 @@ func (s *csiControllerSyncer) ensureProvisionerContainersSpec() []corev1.Contain
 		s.getSidecarImage(config.CSIProvisioner),
 		// TODO: make timeout configurable
 		[]string{"--csi-address=$(ADDRESS)", "--timeout=3m", "--worker-threads=10",
-			"--extra-create-metadata", "--v=5", "--default-fstype=gpfs",
+			"--extra-create-metadata", "--default-fstype=gpfs",
```

Comment:
not sure, if we are still managing the sidecar log level at present from configmap?

Reply:
We are not managing sidecar log level from config map... It is taken from the pod spec. I have removed --v=5 from podspec for driver container alone as it overrides the value which is read as env from configmap